.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig http://EditorConfig.org
+
+# Project Root
+root = true
+
+# Default Code Style
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = tab
```

Comment:
This is the standard in a lot of languages so it should not be touched. 
The best way its to override JS an HTML, CSS and LESS

Ex:
 [*.{less,css}]
indent_size = 2
indent_style = space

[*.js]
indent_size = 2
indent_style = space

[*.hml]
indent_size = 2
indent_style = space